firewalld/config.sls

```diff
@@ -13,7 +13,7 @@ directory_firewalld:
     - require:
       - pkg: package_firewalld # make sure package is installed
     - listen_in:
-      - module: service_firewalld # restart service
+      - service: service_firewalld # restart service
```

Comment:
@amendlik  Mind you either modify the comments (to reflect the new behaviour) or remove them, for consistency?